lib/poteto/config.rb

```diff
@@ -43,6 +49,11 @@ def commit_id=(commit_id)
       @commit_id = commit_id
     end
 
+    def pr_id=(pr_id)
+      @reviewers.each { |r| r.pr_id = pr_id }
+      @pr_id = pr_id
+    end
+
     def deep_symbolize(h)
```

Comment:
Metrics/PerceivedComplexity: Perceived complexity for deep_symbolize is too high. [10/8]